app/app.py

```diff
@@ -10,7 +10,7 @@ def question_router(question_type, question_id):
     seed = request.args.get("seed", time())
     question = get_question(question_type, question_id, seed)
 
-    return question.__dict__
+    return question.json
```

Comment:
Should this be `question.json()`?

Reply:
i.e. actually call the function

Reply:
nah it's a class property, not a function